independent-projects/qute/core/src/main/java/io/quarkus/qute/TemplateInstance.java

```diff
@@ -31,6 +32,11 @@ public interface TemplateInstance {
      */
     String SELECTED_VARIANT = "selectedVariant";
 
+    /**
+     * Attribute key - locale.
+     */
+    String LOCALE = "locale";
```

Comment:
I think that we should reuse this constant for `io.quarkus.qute.i18n.MessageBundles.ATTRIBUTE_LOCALE`, but we should probably keep `MessageBundles.ATTRIBUTE_LOCALE` for backwards compatibility.